### PR TITLE
Created new docker container for ibm-db2 integration

### PR DESCRIPTION
**Pull request by landonc**
## Status
Ready

## Related Content Pull Request
Related PR: [https://github.com/demisto/content/pull/18139](https://github.com/demisto/content/pull/18139)

## Description
Create docker image for the IBM DB2 integration as it required the ibm_db python package.

### CODE REVIEW

**Comment by GuyAfik** [comment]
@wissamg Can you please CR that one? 
Its a new docker image for that [PR](https://github.com/demisto/content/pull/18139)

File: docker/ibm-db2/Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ibm-db = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.8"
```
Comment (wissamg) [inline docker/ibm-db2/Pipfile:12]: "3.10"
```suggestion
python_version = "3.10"
```

File: docker/ibm-db2/Pipfile.lock [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "ac76b76e65db84c04d527864bc45906f633fd1cba8a6facb5f831d93af2f8523"
+        },
+        "pipfile-spec": 6,
+        "requires": {
+            "python_version": "3.8"
```
Comment (wissamg) [inline docker/ibm-db2/Pipfile.lock:8]:
```suggestion
            "python_version": "3.10"
```

**Comment by GuyAfik** [comment]
@landonc can you please re-generate the pipfile and pipfile.lock to be 3.10 based instead of 3.8?

**Comment by landonc** [comment]
> 
My host system is only at python 3.8 and no updates currently available so trying to modify the Pipfile to 3.10 and running pipenv lock fails.  Suggestions?

**Comment by GuyAfik** [comment]
> > 
> 
> My host system is only at python 3.8 and no updates currently available so trying to modify the Pipfile to 3.10 and running pipenv lock fails. Suggestions?

lets setup a call and ill help you with that

**Review by wissamg — APPROVED** [review]

**Pull request merged**